packages/proxy/lib/http/request-middleware.ts

```diff
@@ -44,17 +44,15 @@ const MaybeAttachCrossOriginCookies: RequestMiddleware = function () {
     this.req.isAUTFrame,
   )
 
-  const cookies = this.getCookieJar().getCookies(this.req.proxiedUrl, sameSiteContext)
-  const existingCookies = this.req.headers['cookie'] ? [this.req.headers['cookie']] : []
-  const cookiesToAdd = cookies.map((cookie) => `${cookie.key}=${cookie.value}`)
+  const applicableCookiesInCookieJar = this.getCookieJar().getCookies(this.req.proxiedUrl, sameSiteContext)
```

Comment:
Should `addCookieJarCookiesToRequest` handle pulling the cookie jar cookies?

Reply:
I figured not since it really it just a utility and didn't want to make it state aware.